ActindoWorkFlow/Operation/06_InsertStaticInput.md

```diff
@@ -0,0 +1,63 @@
+# Insert a static input
+
+With the static input function, you can use an input port to pass customer-specific data to a transition, with which you want to process it. Only valid JSON values are accepted, for example text ("text"), numbers ("42") or JSON objects ({"dispatchNote": {"id": 42}}).  
+For example, you want to send offer data to a specific sales channel. Since it is not possible to input a sales channel to a start place and to pass it through the entire workflow, you can input this information to a transition that then passes the sales channel information to an API endpoint.   
+
+The following procedure shows an example of how to edit the static inputs by adding a sales channel information that requires complex JSON code. 
+
+#### Prerequisites
+
+- You have the required rights to edit a workflow. 
+- The transition you want to add has additional input ports.
+
+
+#### Procedure
+
+*Process Orchestration > Workflows > Tab OVERVIEW > Select workflow > Select version*
+
+![Workflow editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+1. Add the desired transition. For detailed information, see [Add a transition](./01_ManageWorkflows.md#add-a-transition).   
+    The transition is added. It has several input ports which are listed in the *Static inputs* section of the transition settings sidebar. 
+
+    ![Input ports](../../Assets/Screenshots/ActindoWorkFlow/Workflows/TransitionInputPorts.png)
+
+2. Check the API used in the API documentation. To do this, select *Dev Tools > API > Select API tab* and select the desired API.    
+    The API documentation is displayed. On the left side, the API list is displayed. The middle column contains the parameters. The right side shows the JSON code of the API parameters. 
+      
+    ![API documentation](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowAPIDocumentation.png "[API documentation]")
+
+    
+3. Check the parameters of the API.    
+    All required parameters are indicated. You must fill the required parameters to ensure a smooth data transfer.
```

Comment:
Required parameters muss man mit einem Wert versehen, sonst wird die API-Call scheitern, daher denke ich "to ensure a smooth data transfer" ist nicht ganz genau.